ext/gmp/gmp.c

```diff
@@ -1886,8 +1902,8 @@ ZEND_FUNCTION(gmp_clrbit)
 	}
 
 	if (index < 0) {
-		php_error_docref(NULL, E_WARNING, "Index must be greater than or equal to zero");
-		RETURN_FALSE;
+		zend_argument_value_error(2, "must be greater than or equal to zero");
```

Comment:
```suggestion
		zend_argument_value_error(2, "must be greater than or equal to 0");
```

there are a few other cases below